test/accessor_tests.cc

```diff
@@ -216,31 +216,32 @@ namespace detail {
 		auto& tm = runtime::get_instance().get_task_manager();
 		auto& rm = runtime::get_instance().get_reduction_manager();
```

Comment:
Hmm... This is unused but doesn't seem to produce a warning? Weird.

Reply:
Weird indeed... removed it nonetheless.